.github/workflows/lint.yml

```
@@ -0,0 +1,34 @@
+name: Lint
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '*.py'
+  pull_request:
+    branches:
+      - master
+      - dev
+    paths:
+      - '*.py'
+
+jobs:
+  flake8_py3:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7.7
+          architecture: x64
+      - name: Checkout PyTorch
+        uses: actions/checkout@master
+      - name: Install flake8
+        run: pip install flake8
+      - name: Run flake8
+        uses: suo/flake8-github-action@releases/v1
+        with:
+          checkName: 'flake8_py3'   # NOTE: this needs to be the same as the job name
```

I see here that we are relaying on a third party to run `flake8`. I don't think this is necessary as we chan make this change to run `flake8`:
```suggestion
      - name: Run flake8
      - run: flake8 .
```

I'm not sure if this works completely but is something similar in order to run the command and not relaying in third parties.